ndt7/results/file.go

```diff
@@ -105,27 +107,39 @@ type savedMeasurement struct {
 	// Measurement is the actual measurement to be saved.
 	Measurement model.Measurement `json:"measurement"`
 }
+*/
```

Comment:
If this code is not needed anymore, can we zap it rather than commenting it out? If there is a reason to keep it commented out, instead, I'd suggest to explain that with a `//` comment before the `/*`.